### DISCUSSION TITLE
使用中遇到的（界面卡死）问题

### DISCUSSION THREAD

**Discussion by Xi0521** [gkd-kit/gkd#296] [General]
1，软件有概率跳过时卡死界面（本人使用b站测试），退出重进多次才会恢复情况，还望优化
2，趣智校园的跳广告并无执行，软件权限全部给予授权的情况也是如此

**Comment by lisonge** [comment]
> 软件有概率跳过时卡死界面

无法复现

---

> 趣智校园的跳广告并无执行

https://github.com/gkd-kit/subscription/issues/new/choose

**Reply by Xi0521** [reply]
谢谢答复，在新版本更新中已解决这个问题，关于软件“趣智校园”，我将通过分享快照给您，希望您有空看看。
祝，每天开森
